ChangeLog.md

```diff
@@ -0,0 +1,22 @@
+# Revision History for Hydra Pay
+
+This project's release branch is `master`. This log is written from the perspective of the release branch: when changes hit `master`, they are considered released.
+
+##  Unreleased
```

Comment:
Should we cut a new release?

Reply:
The goal is a release, are you saying it makes sense to change "unreleased" to "v1.0.0"? @ali-abrar

Reply:
Yes